apps/src/templates/studioHomepages/ResourceCard.story.jsx

```diff
@@ -14,20 +15,6 @@ export default storybook => {
             description="Ask questions about curriculum, share ideas from your lessons, and get help from other teachers"
             buttonText= "Connect Today"
             link= "link to teacher community"
-            isRtl={false}
-          />
-        )
-      },
-      {
-        name: 'tool card - RTL',
-        description: `This is an example tool card with RTL styling.`,
-        story: () => (
-          <ResourceCard
-            title= "Teacher Community"
-            description="Ask questions about curriculum, share ideas from your lessons, and get help from other teachers"
-            buttonText= "Connect Today"
-            link= "link to teacher community"
-            isRtl={true}
```

Comment:
Removed these stories because they're identical to the other two, just with `isRtl={true}`.  Down the road I think we'll maybe want to make it possible to modify the `isRtl` value in the Redux store from the Storybook UI.